### ISSUE TITLE
Recommend: remove 'www' from service domain

### ISSUE THREAD

**Issue by gaswirth** [oslabs-beta/GQL-OSS#68]
Hi there,

Long time listener first time caller (developer friend of @gacetta from [Roundhouse Designs](https://roundhouse-designs.com))

I'd like to submit a recommendation regarding your domain name: Remove the `www` prefix! Many users will run into CORS issues (as I did) today, since some browsers hide the `www` subdomain -- when I added `graphql-oss.io` to my allowed hosts list, it didn't initially work. After a few minutes, I realized that you're forcing a `www` subdomain, which is both outdated and unnecessary, and which can cause complications. When it's available, I always recommend that my own clients use their root domain and strip out the 'www' if possible.

TL;DR Highly recommend you remove the `www` prefix, or at the very least, stop forcing the redirect.
